File: src/test/java/com/tort/mudai/PersonTest.java
package com.tort.mudai;

import com.google.inject.Guice;
import com.google.inject.Injector;
import org.testng.annotations.AfterTest;
import org.testng.annotations.BeforeTest;
import org.testng.annotations.Test;

import static org.testng.Assert.assertNotNull;

@Test
public class PersonTest {
    Injector injector = Guice.createInjector(new MudaiModule());
    Person person = injector.getInstance(Person.class);

    @BeforeTest
    public void setUp() {
    }

    @AfterTest
    public void tearDown() {
    }

    public void create(){
        assertNotNull(person);
    }
}

File: src/test/scala/com/tort/mudai/task/PersonTest.scala
package com.tort.mudai.task

import org.scalatest.FunSuite
import org.scalatest.matchers.ShouldMatchers
import com.google.inject.Guice
import com.tort.mudai.MudaiModule

class PersonTest extends FunSuite with ShouldMatchers {
  val injector = Guice.createInjector(new MudaiModule)
  val person = injector.getInstance(classOf[Person])

  test("save stat") {
    person.viewStat("test stat")
  }
}
